.github/workflows/continuous_integration.yml

```diff
@@ -109,6 +121,16 @@ jobs:
         if: runner.os == 'macOS'
         run: |
           brew update
+          rm /usr/local/bin/2to3
+          rm /usr/local/bin/2to3-3.12
+          rm /usr/local/bin/idle3
+          rm /usr/local/bin/idle3.12
+          rm /usr/local/bin/pydoc3
+          rm /usr/local/bin/pydoc3.12
+          rm /usr/local/bin/python3
+          rm /usr/local/bin/python3-config
+          rm /usr/local/bin/python3.12
+          rm /usr/local/bin/python3.12-config
```

Comment:
It seems `brew install --overwrite python@3.12` is simpler solution. From https://github.com/Homebrew/homebrew-core/issues/173191